src/clients/api/queries/getCurrentVotes.ts

```diff
@@ -0,0 +1,19 @@
+import BigNumber from 'bignumber.js';
+import { XvsVault } from 'types/contracts';
+
+export interface IGetCurrentVotesInput {
+  xvsVaultContract: XvsVault;
+  accountAddress: string;
+}
+
+export type GetCurrentVotesOutput = BigNumber;
+
+const getCurrentVotes = async ({
+  xvsVaultContract,
+  accountAddress,
+}: IGetCurrentVotesInput): Promise<GetCurrentVotesOutput> => {
+  const resp = await xvsVaultContract.methods.getCurrentVotes(accountAddress).call();
+  return new BigNumber(resp).div(1e18);
```

Comment:
You can use `convertWeiToCoins` to make this operation, this way the XVS decimals aren't hard-coded.